### PR TITLE
easy install for kubernetes

### PR DESCRIPTION
**Pull request by kramasamy**

### CODE REVIEW

File: deploy/kubernetes/helm/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+1. You can check the status of HDFS by running this command:
```
Comment (cckellogg) [inline deploy/kubernetes/helm/templates/NOTES.txt]: Do we need this file?

File: deploy/kubernetes/helm/templates/tools.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    metadata:
+      labels:
+        app: {{ template "heron.fullname" . }}-tools
+        release: {{ .Release.Name }}
+    spec:
+      initContainers:
+        # Wait until the zookeeper pods are up and running
+        - name: {{ template "heron.fullname" . }}-init-container
+          image: {{ .Values.image.repository }}/{{ .Values.image.tag }}
+          command: ['sh', '-c', 'while true; do status=$(echo ruok | nc {{ .Release.Name }}-zookeeper 2181); if [ "$status" = "imok" ]; then break;  fi;  sleep 20;  done']
```
Comment (cckellogg) [inline deploy/kubernetes/helm/templates/tools.yaml]: should we make this a for loop so we fail after sometime?

File: deploy/kubernetes/helm/templates/apiserver.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+          command: ["sh", "-c"]
+          args:
+            - >-
+              heron-apiserver
+              --base-template kubernetes
+              --cluster {{ .Release.Name }}
+              -D heron.statemgr.connection.string={{ .Release.Name }}-zookeeper:2181
+              -D heron.kubernetes.scheduler.uri=http://localhost:8001
+              -D heron.executor.docker.image={{ .Values.image.repository }}/{{ .Values.image.tag }}
+              -D heron.class.uploader=com.twitter.heron.uploader.dlog.DLUploader
```
Comment (cckellogg) [inline deploy/kubernetes/helm/templates/apiserver.yaml]: Should there be a bookkeeper dependency in the chart?

File: deploy/kubernetes/helm/templates/NOTES.txt (hunk 1 above)
Comment (kramasamy) [reply deploy/kubernetes/helm/templates/NOTES.txt]: Yes indeed - it gives some overview of what has been installed.

**Pull request merged**